### PR TITLE
Fix Docker Compose build errors

### LINKED ISSUE #683: Docker Compose build failing
**Describe the bug**
The Docker Compose build section of the README fails with the following error (with `-V` flag passed to `jekyll serve`):

```
github-wiki-theme_1  |          Rendering: docs/es/achievement.md
github-wiki-theme_1  |   Pre-Render Hooks: docs/es/achievement.md
github-wiki-theme_1  |   Rendering Markup: docs/es/achievement.md
github-wiki-theme_1  | Invalid Markdown processor given: GFM
github-wiki-theme_1  |                     Available processors are: rdiscount, kramdown, redcarpet
github-wiki-theme_1  |   Conversion error: Jekyll::Converters::Markdown encountered an error while converting 'docs/es/achievement.md':
github-wiki-theme_1  |                     Bailing out; invalid Markdown processor.
github-wiki-theme_1  |              ERROR: YOUR SITE COULD NOT BE BUILT:
github-wiki-theme_1  |                     ------------------------------------
github-wiki-theme_1  |                     Bailing out; invalid Markdown processor.
wiki_github-wiki-theme_1 exited with code 1
```

**To reproduce**
1. Install `docker-compose`
2. Run `docker-compose up github-wiki-theme`

**Expected behavior**
The wiki should compile and be accessible on `localhost:4000`.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version: 96.0.2

**Additional context**
I have managed to get the build to pass by adding `jekyll-remote-theme` and removing GFM from Jekyll's config, however the wiki is still inaccessible after this because Jekyll resets the URL in the address bar to 0.0.0.0 in development mode. Setting `JEKYLL_ENV=production` makes the wiki accessible, however the links in the sidebar seem to be broken.

### PR DESCRIPTION
**Pull request by aaronhooper**
### Description
I have modified `_config.yml` and `docker-compose.yml` to fix the build errors as described in #683.

### Related Issue
Fixes #683

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -58,7 +58,6 @@ google_cse_token:
 # git-wiki includes some internal themes that you can choose
 # check _layouts folder
 #
-markdown: GFM
```
Comment (Kitzunu) [inline _config.yml:61]: no

**Comment by Kitzunu** [comment]
also should probably be fixed here https://github.com/Drassil/git-wiki

File: _config.yml (hunk 1 above)
Comment (aaronhooper) [reply _config.yml:61]: [The Jekyll docs](https://jekyllrb.com/docs/configuration/markdown/#kramdown-processor) mention that Kramdown and GFM are used by default and there is no need to specify it. What exactly is the problem with this?

Comment (Kitzunu) [reply _config.yml:61]: We've had issues in the past. But I am not familiar with it ngl.

But as my request below, this should be fixed in the other repo to then be cherrypicked here

**Pull request closed without merging**